test/integration/checkout_int_test.go

```diff
@@ -71,7 +71,7 @@ func (suite *CheckoutIntegrationTestSuite) TestCheckoutWithFlags() {
 		e2e.WithArgs("checkout", "ActiveState-CLI/Python3#6d9280e7-75eb-401a-9e71-0d99759fbad3", "--path", python3Dir),
 		e2e.AppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"),
 	)
-	cp.ExpectExitCode(0)
+	cp.Expect("Checked out")
```

Comment:
Can both the expectexitcode and this call be used successfully?

Reply:
Yes, it looks like that works.